### DISCUSSION TITLE
Neon Helm Chart

### DISCUSSION THREAD

**Discussion by balaatsky** [neondatabase/neon#5656] [Show and tell]
I have put together a helm chart for deploying neon on k8s.
Repo: [Neon Helm Chart Repo](https://github.com/itsbalamurali/neon-helm)
Also working on an operator to automate the operations and include a cloud control plane alternative.
Please feel free to contribute.

**Comment by itsbalamurali** [comment]
Neon k8s Operator: https://github.com/itsbalamurali/neon-operator
